### PR TITLE
#540 call create outlook email enhancement

### PR DESCRIPTION
**Pull request by MDR22**
Updated the function to include functionality noted in issue, along with ability to add follow-up flag with reminder, and add multiple attachments. I included comments on the flag functionality and multiple attachments commits with some consideration for possible adjustments and/or feedback.

### CODE REVIEW

File: misc/desert-island-menu.vbs
Comment (MDR22) [inline misc/desert-island-menu.vbs:1]: Triggered script error at Line 47, Column 5. The function parameters all appear to be correct - guessing this is due to desert island functionality?

File: misc/tags-main-menu.vbs
Comment (MDR22) [inline misc/tags-main-menu.vbs:1]: Triggered a script error at Line 87, Column 1 - object not a collection. The function parameters appear to be correct. Similar to other comment on MISC script in guessing this relates to desert island script functionality.

File: utilities/va-verification-request.vbs
Comment (MDR22) [inline utilities/va-verification-request.vbs:1]: The comment in the script indicates that the email will not send automatically thereby allowing for review but the function parameter is set to send the email automatically. Not sure if this should be changed.

Comment (MDR22) [reply utilities/va-verification-request.vbs:1]: Removed the comment as it was incorrect

File: MASTER FUNCTIONS LIBRARY.vbs [hunk 1]
```diff
@@ -5547,13 +5547,22 @@ FUNCTION create_outlook_appointment(appt_date, appt_start_time, appt_end_time, a
 	objAppointment.Save												'Saves the appointment
 END FUNCTION
 
-Function create_outlook_email(email_recip, email_recip_CC, email_subject, email_body, email_attachment, send_email)
-'--- This function creates a an outlook appointment
-'~~~~~ (email_recip): email address for recipeint - seperated by semicolon
-'~~~~~ (email_recip_CC): email address for recipeints to cc - seperated by semicolon
+Function create_outlook_email(email_from, email_recip, email_recip_CC, email_recip_bcc, email_subject, email_importance, include_flag, email_flag_text, email_flag_days, email_flag_reminder, email_flag_reminder_days, email_body, include_email_attachment, email_attachment_array, send_email)
```
Comment (IlseFerris) [inline MASTER FUNCTIONS LIBRARY.vbs:5550]: Excellent function parameter notes. Future us will be very happy :)

**Review by IlseFerris — APPROVED** [review]
I double checked all the parameters on each function call, and it all looks great. I approve thee!

**Pull request merged**